assets/scss/partials/footer.scss

```diff
@@ -1,20 +1,117 @@
+.main-footer {
+  display: grid;
+  grid-template-columns: 10% auto 10%;
+  grid-template-rows: repeat(1, 15rem);
+  grid-column-gap: 0px;
+  grid-row-gap: 0px;
+}
+
+.footer-left-whitespace{
+  grid-area: 1 / 1 / 2 / 2;
+}
+
+.footer-right-whitespace {
+  grid-area: 1 / 3 / 2 / 4;
+}
+
+.footer-center-content {
+  grid-area: 1 / 2 / 2 / 3;
+}
+
 .footer-bottom-wrap{
+  display: grid;
+  grid-area: 2 / 1 / 3 / 4;
```

Comment:
Awesome! I see you're taking full advantage of the power of grid! If you end up using a lot of `grid-area`s like this, you might want to consider naming grid areas and assigning elements to those areas. Makes the CSS easier to handle and more readable.